bigquery/api/resources/data.csv

```diff
@@ -1 +0,0 @@
-Gandalf,2000,140.0,1
```

Comment:
You might want to double-check that none of the cloud-client sample tests use these test data files.

Reply:
The snippets tests are passing and I didn't see any usage of data files, which prompted me to also delete the unused cloud-client data files.